internal/pkg/plugin/manager.go

```diff
@@ -68,13 +71,21 @@ func GetManager() *Manager {
 	return _singleton
 }
 
+var ProfileCleaner interface {
```

Comment:
let's move the ProfileCleaner into the `_RegisterShutdownSignalHandlers` so that it's not global